pkg/storage/bucket/hedged_reqeust_test.go

```diff
@@ -0,0 +1,36 @@
+package bucket
```

Comment:
there is a typo in the name of the file. it should be hedged_request_test.go

Reply:
Thanks for catching it.. Fixed!